### ISSUE TITLE
Cannot find module '.../node_modules/@loaders.gl/flatgeobuf/node_modules/flatgeobuf/lib/mjs/generic/feature'

### ISSUE THREAD

**Issue by aspyk** [visgl/loaders.gl#2857]
I have a strange behavior during `vitest` unit test with my typescript app using `@loaders.gl/flatgeobuf": "^4.0.4"`:
```
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯ Failed Suites 1 ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯

 FAIL  src/[...].test.ts [ [...].ts ]
Error: Cannot find module '/home/[...]/node_modules/@loaders.gl/flatgeobuf/node_modules/flatgeobuf/lib/mjs/generic/feature' imported from /home/[...]/node_modules/@loaders.gl/flatgeobuf/dist/lib/parse-flatgeobuf.js
Did you mean to import flatgeobuf/node_modules/flatgeobuf/lib/mjs/generic/feature.js?
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

```
The unit test does not call directly a function of the flatgeobuf lib, it just imports a module having `import { FlatGeobufLoader } from "@loaders.gl/flatgeobuf";` in the header.

However, when I serve my App (used in Electron as standalone app), everything works fine.

Do you have any idea of what could be wrong ?

**Comment by ibgreen** [comment]
The flatgeobuf module is packaged in a way that is not compatible with all JS tooling. 

We have landed a fix for this in the latest 4.1.0-alpha.8 release, and we are also seeing if we can upstream some fixes but that might take more time.

**Comment by aspyk** [comment]
Ok thank you for your quick reply ! 
I'll check out this fix, and since my app works fine anyway, it's always good news to know there's something in the works, no matter how long it takes :)
Thanks again !
